ovn-tester/ovn_tester.py

```diff
@@ -19,34 +19,34 @@
 
 
 GlobalCfg = namedtuple(
-    'GlobalCfg', ['log_cmds', 'cleanup', 'run_ipv4', 'run_ipv6']
+    "GlobalCfg", ["log_cmds", "cleanup", "run_ipv4", "run_ipv6"]
```

Comment:
This was kind of on purpose and to avoid large diffs, we enforced black formatting but **without** string quotes or prefixes normalization:
https://github.com/ovn-org/ovn-heater/pull/104/commits/9c3e00765f8433c5e0d7e2fb1d28fd7c73d1cc16

However, if I apply string quotes normalization to the whole repo I get:
```
$ git diff --stat
 ovn-fake-multinode-utils/generate-hosts.py   |  46 ++++++++++++-------------
 ovn-fake-multinode-utils/get-config-value.py |   4 +--
 ovn-fake-multinode-utils/process-monitor.py  |  20 +++++------
 ovn-fake-multinode-utils/translate_yaml.py   |  20 +++++------
 ovn-tester/ovn_context.py                    |  12 +++----
 ovn-tester/ovn_ext_cmd.py                    |  22 ++++++------
 ovn-tester/ovn_load_balancer.py              |  24 ++++++-------
 ovn-tester/ovn_sandbox.py                    |  30 ++++++++--------
 ovn-tester/ovn_stats.py                      |  42 +++++++++++-----------
 ovn-tester/ovn_tester.py                     | 110 +++++++++++++++++++++++++++++-----------------------------
 ovn-tester/ovn_utils.py                      | 130 ++++++++++++++++++++++++++++++++++----------------------------------
 ovn-tester/ovn_workload.py                   | 352 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++---------------------------------------------------------------------------------------------
 ovn-tester/tests/cluster_density.py          |  16 ++++-----
 ovn-tester/tests/density_heavy.py            |  28 +++++++--------
 ovn-tester/tests/density_light.py            |  16 ++++-----
 ovn-tester/tests/netpol.py                   |  14 ++++----
 ovn-tester/tests/netpol_cross_ns.py          |  16 ++++-----
 ovn-tester/tests/netpol_large.py             |   2 +-
 ovn-tester/tests/netpol_multitenant.py       |  34 +++++++++---------
 ovn-tester/tests/netpol_small.py             |   2 +-
 ovn-tester/tests/service_route.py            |  26 +++++++-------
 utils/latency.py                             |  34 +++++++++---------
 utils/process-stats.py                       |  42 +++++++++++-----------
 23 files changed, 521 insertions(+), 521 deletions(-)
 ```

It doesn't seem that extreme, so maybe it's acceptable to just do this all in a single commit now that we're refactoring most of the tester.

In any case, we should at least enforce this for new changes so please also update the `black` invocation in the [GitHub action worflow definition ](https://github.com/ovn-org/ovn-heater/blob/648b221349f5bd2d8f51191d7443ad216c54a51c/.github/workflows/test.yml#L33)and remove the `-S`.

Reply:
Just adding my opinion on this, I don't think it's a good idea to do this change. It bloats git history a bit and doesn't provide any value IMO. From the PEP 8 perspective we are fine with both:

```
In Python, single-quoted strings and double-quoted strings are the same. This PEP does not make a
recommendation for this. Pick a rule and stick to it. When a string contains single or double quote characters,
however, use the other one to avoid backslashes in the string. It improves readability.
```

Reply:
OTOH, we're changing a lot of the code base so if we want to make such a change, now is the moment IMO.

Reply:
The formatting was applied to make the CI happy for file changed, and for some reason I picked up on the project using `black -l 79` but missed the `-S` when applying it.

I'm fine with redoing those commits with `-S` applied and my personal style is to use single quotes for strings as well (`''`).

To avoid this in the future, what do you think about adding a top level `pyproject.toml` file with [black configuration](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#configuration-via-a-file)?

For example
```toml
[tool.black]
line-length = 79
skip-string-normalization = 1
```